region/maridia/outer/Crab Hole.json

```diff
@@ -657,6 +772,183 @@
       ],
       "devNote": "FIXME: It is possible to crab climb with nothing but ice and walljumps."
     },
+    {
+      "link": [2, 1],
+      "name": "Cross Room Spring Ball Bounce Unmorph (Speedbooster)",
+      "notable": true,
+      "reusableRoomwideNotable": "Crab Hole Cross Room Jump Morph",
+      "entranceCondition": {
+        "comeInWithSpringBallBounce": {
+          "remoteAndLandingMinTiles": [[3, 1]],
+          "movementType": "uncontrolled"
+        }
+      },
+      "requires": [
+        "canCrossRoomJumpIntoWater",
+        "canMomentumConservingMorph",
+        "canDoubleSpringBallJumpMidAir"
```

Comment:
It's only with SpeedBooster equipped that the springball jump can be needed. Speeds $1.2 and $1.3 are the ones that require a spring ball jump if you don't get the last-frame bounce. Speed $1.2 is the max speed you can get with a jump from a 3-tile runway with 1 closed end. Requiring a last-frame bounce seemed like it could increase the difficulty too much; though maybe this could also be handled through randomizer-side leniency on the runway speed. And in any case, it would still be an option to do the spring ball jump even if it's not included as a requirement, so I guess it doesn't matter too much :shrug: